lib/src/base-styles/_nx-lists.scss

```diff
@@ -133,104 +130,64 @@
   }
 
   .nx-list__item {
-    @include semi-bold();
-
-    color: $nx-text-color-light;
-
     &.selected {
-      background-color: $nx-grey-3;
-      color: $nx-text-color;
+      background-color: #d3e6ff;
       pointer-events: none;
     }
 
-    &:not(.nx-list__item--empty):not(.nx-disabled):not([disabled]) {
+    &:not(.nx-list__item--empty):not(.disabled):not([disabled]),
+    &:not(.nx-list__item--error) {
       &:hover, &:focus {
-        background-color: $nx-color-blue-96;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
-        color: $nx-text-color;
+        background-color: #f1f3fa;
         cursor: pointer;
         outline-style: none; // removes :focus outline
 
         .fa-chevron-right {
-          color: $nx-color-blue-41;
+          color: #045dd1;
         }
       }
 
       &:active {
-        background-color: $nx-grey-1;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
+        background-color: #d3e6ff;
       }
     }
 
     &.nx-list__item--empty {
       @include regular();
 
-      border-top: $nx-border; // assume it's always the first item
       color: $nx-text-color-light;
     }
 
-    &.nx-error {
-      color: $nx-text-color-error;
-
-      &:not(.nx-disabled):not([disabled]) {
-        border: $nx-border-error;
-
-        &:hover, &:focus, &:active {
-          background-color: $nx-fail-color-bg;
-          border: $nx-border-error;
-          color: $nx-text-color-error;
-          cursor: default;
-        }
-      }
-    }
-
     .nx-list__subtext {
       pointer-events: none; // Fix for IE11 :active state display
     }
 
-    &.nx-list__item--disabled {
+    &.disabled {
       color: $nx-text-color-disabled;
-      cursor: default;
 
       &:hover {
-        border-top: 1px solid transparent;
-        border-bottom: $nx-border-row;
+        background-color: transparent;
+        cursor: default;
       }
     }
   }
 
-  .fa-chevron-right {
-    margin-right: 3px;
-    margin-top: 1px;
+  .nx-icon {
+    align-self: center;
+    grid-column: 2;
+    grid-row: 1;
+    justify-self: end;
   }
 }
 
-/*
- * Definition Lists
- *
- * Typically use <dl>
- */
-
-.nx-list--definition-list {
-  box-sizing: border-box;
-
-  display: flex;
-  flex-wrap: wrap;
-
-  .nx-list__item {
-    @include regular();
-
-    display: inline-block;
-    width: 75%;
-
-    white-space: normal;
-  }
+.nx-list-actions {
+  @include container-horizontal;
 
-  .nx-list__item--label {
-    @include semi-bold();
+  grid-column: 2;
+  grid-row: 1;
+  justify-self: end;
 
-    width: 25%;
+  .nx-btn {
+    margin: 0 0 0 $nx-spacing-xs;
```

Comment:
This shouldn't need to be done here, but rather a global solution should exist.  I think inter-button spacing is actually something we haven't worked out in the redesign in the general case yet, believe it or not.  The way it's supposed to work, and used to work before 1.0, is that buttons should have left and right margin by default, and a button that immediately follows another button should have its left margin removed.  And then of course they should typically be used in a container that uses the `container-horizontal` mixin.

Reply:
> This shouldn't need to be done here, but rather a global solution should exist. I think inter-button spacing is actually something we haven't worked out in the redesign in the general case yet, believe it or not. The way it's supposed to work, and used to work before 1.0, is that buttons should have left and right margin by default, and a button that immediately follows another button should have its left margin removed. And then of course they should typically be used in a container that uses the `container-horizontal` mixin.

@dturner32 what is the default spacing between adjacent buttons?

Reply:
> > This shouldn't need to be done here, but rather a global solution should exist. I think inter-button spacing is actually something we haven't worked out in the redesign in the general case yet, believe it or not. The way it's supposed to work, and used to work before 1.0, is that buttons should have left and right margin by default, and a button that immediately follows another button should have its left margin removed. And then of course they should typically be used in a container that uses the `container-horizontal` mixin.
> 
> @dturner32 what is the default spacing between adjacent buttons?

24px

Reply:
> > > This shouldn't need to be done here, but rather a global solution should exist. I think inter-button spacing is actually something we haven't worked out in the redesign in the general case yet, believe it or not. The way it's supposed to work, and used to work before 1.0, is that buttons should have left and right margin by default, and a button that immediately follows another button should have its left margin removed. And then of course they should typically be used in a container that uses the `container-horizontal` mixin.
> > 
> > 
> > @dturner32 what is the default spacing between adjacent buttons?
> 
> 24px

I created this ticket to change buttons globally: https://issues.sonatype.org/browse/RSC-287

Here's a slight improvement that will match the solution that we'll likely use for the ticket above: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/eb551dd6441bce099dfa54552ce4f95752fd167b

Note that the button spacing here is 8px, a custom value

Reply:
> Note that the button spacing here is 8px, a custom value

I wonder if that's really supposed to be custom to this area or if it should always be 8 between icon buttons, or something.